apps/studio/src/components/export/ExportNotification.vue

```diff
@@ -37,10 +45,13 @@ export default {
       }
       this.exporter.abort();
       this.notification.close();
-      this.$noty.error(`${this.exporter.table.name} export aborted`);
+      const exportName = this.exporter.table ? this.exporter.table.name : this.query_name;
+      this.$noty.error(`${exportName} export aborted`);
```

Comment:
I think this is where the `undefined` comes

Reply:
Good catch! this is fixed in 980017d